objects_and_classes/homework/homework.py

```diff
@@ -1,60 +1,237 @@
+from constants import CARS_TYPES, CARS_PRODUCER, TOWNS
+from typing import List
+
+import string
+import random
+import uuid
+
+
 """
 Вам небхідно написати 3 класи. Колекціонери Гаражі та Автомобілі.
 Звязкок наступний один колекціонер може мати багато гаражів.
 В одному гаражі може знаходитися багато автомобілів.
-
 Автомобіль має наступні характеристики:
     price - значення типу float. Всі ціни за дефолтом в одній валюті.
     type - одне з перечисленних значеннь з CARS_TYPES в docs.
     producer - одне з перечисленних значеннь в CARS_PRODUCER.
     number - значення типу UUID. Присвоюється автоматично при створенні автомобілю.
     mileage - значення типу float. Пробіг автомобіля в кілометрах.
-
-
     Автомобілі можна перівнювати між собою за ціною.
     При виводі(logs, print) автомобілю повинні зазначатися всі його атрибути.
-
     Автомобіль має метод заміни номеру.
     номер повинен відповідати UUID
-
 Гараж має наступні характеристики:
-
     town - одне з перечислениз значеннь в TOWNS
     cars - список з усіх автомобілів які знаходяться в гаражі
     places - значення типу int. Максимально допустима кількість автомобілів в гаражі
     owner - значення типу UUID. За дефолтом None.
-
-
     Повинен мати реалізованими наступні методи
-
     add(car) -> Добавляє машину в гараж, якщо є вільні місця
     remove(cat) -> Забирає машину з гаражу.
     hit_hat() -> Вертає сумарну вартість всіх машин в гаражі
-
-
 Колекціонер має наступні характеристики
     name - значення типу str. Його ім'я
     garages - список з усіх гаражів які належать цьому Колекціонеру. Кількість гаражів за замовчуванням - 0
     register_id - UUID; Унікальна айдішка Колекціонера.
-
     Повинні бути реалізовані наступні методи:
     hit_hat() - повертає ціну всіх його автомобілів.
     garages_count() - вертає кількість гаріжів.
     сars_count() - вертає кількість машиню
     add_car() - додає машину у вибраний гараж. Якщо гараж не вказаний, то додає в гараж, де найбільше вільних місць.
     Якщо вільних місць немає повинне вивести повідомлення про це.
-
     Колекціонерів можна порівнювати за ціною всіх їх автомобілів.
 """
 
 
 class Cesar:
-    pass
 
+    def __init__(self, name, garages=None):
+        self.name = name
+        self.register_id = uuid.uuid4().hex
+        self.garages = garages if garages is not None else [Garage(owner=self.register_id) for _ in range(random.randrange(1, 3))]
```

Comment:
ідея хороша, але реалізацію варто покращити. Наприклад те, що після `else` загорнути в якийсь метод і там його викликати 
```suggestion
        self.garages = garages if garages else self.generate_garages(how_many)
```

Reply:
Зробив черещ статік метод

Reply:
:+1: